apps/consumer-client/src/pages/examples/gpc-proof.tsx

```diff
@@ -195,6 +214,11 @@ export function openGPCPopup(
       value: externalNullifier,
       userProvided: false
     },
+    membershipLists: {
+      argumentType: ArgumentTypeName.String,
+      value: membershipLists !== undefined ? membershipLists : undefined,
```

Comment:
I think the ?: here is a nop?  emptyStrToUndefined already did its work, and all this line is doing is turning undefined into undefined.